Kudu.Core/Deployment/Generator/ExternalCommandFactory.cs

```diff
@@ -121,7 +122,14 @@ private string StarterScriptPath
         {
             get
             {
-                return Path.Combine(_environment.ScriptPath, StarterScriptName);
+                if (OSDetecter.IsOnWindows())
+                {
+                    return Path.Combine(_environment.ScriptPath, StarterScriptName);
+                }
+                else
+                {
+                    return "/bin/bash";
```

Comment:
This could use a comment. It's not obvious why a custom starter.cmd script on Windows becomes just 'bash' on Linux.